docs/core/utils.rst

```diff
@@ -4,6 +4,8 @@
 Utils
 #####
 
+The ``parse_resource`` function provides a way to break a resource 
+into its bucket and collection id parts
```

Comment:
Why the ``parse_resource()`` would be documented differently? This is not necessary IMO